avaframe/in3Utils/cfgUtils.py

```diff
@@ -264,17 +266,31 @@ def writeCfgFile(avaDir, module, cfg, fileName=''):
     # write to file
     if fileName != '':
```

Comment:
change this to `if filename`
this is false if filename = '' and true if not

Reply:
changed